src/utils/webrtc/analyzers/PeerConnectionAnalyzer.js

```diff
@@ -737,14 +779,32 @@ PeerConnectionAnalyzer.prototype = {
 		console.debug(tag + 'Packets per second: ' + this._packetsPerSecond[kind].toString())
 		console.debug(tag + 'Round trip time: ' + this._roundTripTime[kind].toString())
 		console.debug(tag + 'Timestamps: ' + this._timestampsForLogs[kind].toString())
+
+		this._logRtcStats(kind)
 	},
 
+	_logRtcStats(kind) {
+		const tag = this._getLogTag(kind)
+
+		for (const rtcStats of this._rtcStats[kind]) {
+			if (!rtcStats.length) {
+				console.debug(tag + 'no matching type')
+				continue
+			}
+
+			for (const rtcStat of rtcStats) {
+				console.debug(tag + JSON.stringify(rtcStat))
+			}
```

Comment:
Now sure if it's not going to create a mess in logs. Maybe add array indices to it (so it's visible there are 1..7 stats logged)

I also tried `console.debug(tag, rtcStats.map(stat => JSON.stringify(stat)))`, IMO less readable, but at least compact and provides a better context

Reply:
Added an index for both the group of stats as well as the index in the group itself, but kept the existing code style.